src/hooks/useDocumentDrives.ts

```diff
@@ -12,17 +12,17 @@ const documentDrivesAtom = atom(
     new Map<IDocumentDriveServer, DocumentDriveDocument[]>(),
 );
 
-function driveToHash(drive: DocumentDriveDocument): string {
+export function documentToHash(drive: Document): string {
```

Comment:
These methods provide a faster way to compare if two documents are equal. They compare the hash of the latest operation of each scope.
We could use this to memoize components that receive a document as a prop.